include/fty_common_db_asset.h

```diff
@@ -178,6 +178,17 @@ namespace DBAssets {
                    const std::string &value,
                    uint32_t       element_id);
 
+// count_asset_keytag: how many times is gived a couple keytag/value
+// from an asset in t_bios_asset_ext_attributes
+// return -1 in case of error
+//         0 if there is no such pair in db yet
+//         otherwise number of such pairs
```

Comment:
The table `t_bios_asset_ext_attributes` has a unique key requirement on (`keytag`, `id_asset_element`) so there can be at most one given keytag for an asset. Therefore, there can be at most one such pair and the function can return only -1, 0 or 1 in practice.

The name `count_asset_keytag` is misleading since the "count" is at most one and the function checks if the property equals a given value. A more appropriate name might be `has_asset_keytag_value`.

Reply:
Take into account: rename count_asset_keytag to has_asset_keytag_value